static/app/components/dropdownMenu.tsx

```diff
@@ -25,8 +26,10 @@ export type GetMenuArgs<E extends Element> = {
   itemCount?: number;
 };
 
-// Props for the "actor" element of `<DropdownMenu>`
-// This is the element that handles visibility of the dropdown menu
+/**
+ * Props for the "actor" element of `<DropdownMenu>`
+ * This is the element that handles visibility of the dropdown menu
+ */
```

Comment:
👏  Thanks for fixing the comment type. This allows the language server to correctly interpret these comments and show them as help on the types!